test/unit/validators/url_test.exs

```diff
@@ -68,11 +68,8 @@ defmodule EctoCommons.URLValidatorTest do
     {"http://", []},
     {"http:///a", []},
     {"foo.com", []},
-    {"rdar://1234", []},
-    {"h://test", []},
     {":// should fail", []},
     {"://", []},
-    {"ftps://foo.bar/", []},
```

Comment:
Maybe move them to valid_urls check so that it's explicit they are accepted now ? :)

Reply:
Done :)